### ISSUE TITLE
TestLogsFollowGoroutinesWithStdout is flaky

### ISSUE THREAD

**Issue by ahmetalpbalkan** [docker/docker#13233]
New `TestLogsFollowGoroutinesWithStdout` test randomly fails, we just rebuilt and it went away.

```
FAIL: docker_cli_logs_test.go:400: DockerSuite.TestLogsFollowGoroutinesWithStdout

docker_cli_logs_test.go:439:
   c.Assert(nroutines, check.Equals, getNGoroutines())
... obtained int = 34
... expected int = 32
```
cc: @cpuguy83

**Comment by coolljt0725** [comment]
Maybe the timeout ` t := time.After(5 * time.Second)` should be longer to avoid randomly fails.

**Comment by cpuguy83** [comment]
That's one way... I suppose in a failing case waiting some extra time is not a huge deal.

**Comment by unclejack** [comment]
This is still a problem.

**Comment by runcom** [comment]
closed by #13387
